### PR TITLE
[Component with its own class name as service type] LPS-195840 Refctor Registry classes from Asset, Headless and Saml module

### PR DESCRIPTION
**Pull request by Valmir-unicap**

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:help

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/action/util/AssetEntryActionRegistryUtil.java [hunk 1]
```diff
@@ -14,17 +14,18 @@
...
-import org.osgi.service.component.annotations.Activate;
-import org.osgi.service.component.annotations.Component;
+import org.osgi.framework.FrameworkUtil;
 
 /**
  * @author Jürgen Kappler
  */
-@Component(service = AssetEntryActionRegistry.class)
-public class AssetEntryActionRegistry {
+public class AssetEntryActionRegistryUtil {
```
Comment (rafaprax) [inline modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/action/util/AssetEntryActionRegistryUtil.java:24]: inline this class into its only usage

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:sf

File: modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java [hunk 2]
```diff
@@ -2226,6 +2230,19 @@ private List<AssetEntry> _filterAssetTagNamesAssetEntries(
 		return filteredAssetEntries;
 	}
 
+	private List<AssetEntryAction<?>> _getAssetEntryActionsRegistry(
+		String className) {
```
Comment (rafaprax) [inline modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java:2234]: inline this method into the only usage

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Review by rafaprax — APPROVED** [review]

**Comment by Valmir-unicap** [comment]
PR sent: https://github.com/liferay-core-infra/liferay-portal/pull/5631

**Pull request closed without merging**